src/App.js

```diff
@@ -1,19 +1,14 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Router, Route, browserHistory } from 'react-router'
+import Home from './components/Home.react.js';
```

Comment:
It's just a placeholder for now. I just wanted to put something in for the router boilerplate.